include/tvm/runtime/disco/session.h

```diff
@@ -195,6 +195,18 @@ class SessionObj : public Object {
    * \param host_array The array to be copied to worker-0
    * \param remote_array The NDArray on worker-0
    */
+
+  DRef CopyToWorker0Wrapper(const NDArray& host_array) { 
```

Comment:
To minimize the interface, let us consider only expose remote.Empty.

Then move the wrapper to the upper level applications

Reply:
`Empty` is exposed on python: https://github.com/apache/tvm/blob/unity/python/tvm/runtime/disco/session.py#L117-L142.

Shall we migrate some of the python pieces to C++?

Reply:
If we want to expose in c++, that's only 2 lines of code. So I don't think it's necessary.